packages/fxa-settings/src/components/CardHeader/index.stories.tsx

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from 'react';
-import CardHeader from '.';
+import CardHeader, { CardHeaderProps } from '.';
```

Comment:
Is `CardHeaderProps` used anywhere in this file?

Reply:
It isn't, thanks! 

I could have used `props: CardHeaderProps` instead of `props: React.ComponentProps<typeof CardHeader>`... the latter is nice when defining certain `Subject`s and it prevents an extra export/import and name change if we ever change it but we don't have to set it as a pattern everywhere. I'll just remove this import. 🤷‍♀️